### ISSUE TITLE
build issue, can't resolve dependencies

### ISSUE THREAD

**Issue by kongomongo** [SteamGridDB/steamgriddb-manager#114]
using windows 10, node 15.5 x64

>npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: eslint-plugin-react-hooks@4.2.0
npm ERR! node_modules/eslint-plugin-react-hooks
npm ERR!   dev eslint-plugin-react-hooks@"^1.7.0" from the root project
npm ERR!   peer eslint-plugin-react-hooks@"^4 || ^3 || ^2.3.0 || ^1.7.0" from eslint-config-airbnb@18.2.1
npm ERR!   node_modules/eslint-config-airbnb
npm ERR!     dev eslint-config-airbnb@"^18.2.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! dev eslint-plugin-react-hooks@"^1.7.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: eslint@6.8.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0" from eslint-plugin-react-hooks@1.7.0
npm ERR!   node_modules/eslint-plugin-react-hooks
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

**Comment by kongomongo** [comment]
due to 4140c3ab3eb0ad375f61cf925d6963ac39445dc6 ?

**Comment by kongomongo** [comment]
https://github.com/kongomongo/steamgriddb-manager/commit/48cf7f3e217611d1c46a4f8bdf1083c64cc7e94f

**Comment by aranel616** [comment]
That was just added yesterday, try running npm install.

**Comment by aranel616** [comment]
Took a closer look at the log you posted. I'm not sure why that's happening, but there were a lot of updates to dependencies recently. Try removing your node_modules folder, make sure there aren't any local changes to your package lock file, and running `npm install` again.

**Comment by kongomongo** [comment]
I am pretty sure I already did that on two workstations, I'll try again though. You using node 15.5 too?

Will report back

**Comment by kongomongo** [comment]
npm WARN eslint-plugin-react-hooks@1.7.0 requires a peer of eslint@^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.

thats what I get on github without the -lock file

https://pastebin.com/FVX2zJhw

**Comment by aranel616** [comment]
I'm using npm version 6.14.5 and 12.18.2. I tried deleting my `node_modules` directory and was able to run `npm install` with no issues. 

`eslint-plugin-react-hooks@1.7.0` is very out of date, which may have something to do with it. I've updated that package to 4.2.0 in e4e36d3213bc5d5730f74941db520b9efd2ca3b0, and I'm not having any issues. Try updating to the latest commit and see if that works. If it doesn't see if it works with a more recent version of node.
